cypress/integration/findings_spec.js

```diff
@@ -3,21 +3,221 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { PLUGIN_NAME } from '../support/constants';
 import {
-  PLUGIN_NAME,
-  TWENTY_SECONDS_TIMEOUT,
   TEST_INDEX,
   TEST_DETECTOR,
   TEST_DOCUMENT,
-} from '../support/constants';
+  TEST_FIELD_MAPPINGS,
+} from '../fixtures/constants';
 
 describe('Findings', () => {
   before(() => {
-    // delete any existing indices
+    // Delete any existing indices
     cy.deleteAllIndices();
+
+    // Create test index
+    cy.createIndex('cypress-test-windows', TEST_INDEX);
+
+    // Create detector manually
+
+    // Visit Detectors page
+    cy.visit(`${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/detectors`);
+
+    cy.wait(10000);
```

Comment:
isn't 10000 too long?

Reply:
This is to let the currently appearing error toasts disperse.  That is on my tasks list.  will update when errors under control

Reply:
Do the toasts prevent clicking even when `force` is `true`? If not, it may not be necessary to wait for the toasts to timeout.

Reply:
lowering to 5000 still need to allow initial page load.